fs/dyndev.h

```diff
@@ -0,0 +1,31 @@
+#ifndef FS_DYN_DEV_H
+#define FS_DYN_DEV_H
+
+// dyn_dev's are dynamically added devices (character only for now)
+// with custom dev_ops assigned
+// It's useful to add new device "drivers" in runtime (for example,
+// devices only present on some platforms)
+
+// (int)('D'+'Y') == 157
+#define DYN_DEV_MAJOR 157
```

Comment:
On Linux, major number 157 is reserved for a Specialix RIO serial card. Major numbers 234-254 are reserved for dynamic assignment, so we should use one of those.

Reply:
Linux assigns them starting from 254 and downward -- should I go upward or downward (234 or 254)?

Reply:
Doesn't really matter